packages/devtools/src/commands/isElementSelected.js

```diff
@@ -0,0 +1,9 @@
+import getElementAttribute from './getElementAttribute'
+import getElementTagName from './getElementTagName'
+
+export default async function isElementSelected ({ elementId }) {
+    const tagName  = await getElementTagName.call(this, { elementId })
+    const name = tagName === 'option' ? 'selected' : 'checked'
+    const isSelected = await getElementAttribute.call(this, { elementId, name })
```

Comment:
I think we should use `getElementProperty` here instead of get element attribute.
Select option doesn't necessarily has `selected` attributed even if selected, while property `selected` always reflects actual state